Unable to create build on linux (wrong UnityEngine.dll path)

```
error CS0006: Metadata file `/opt/Unity/Editor/Unity/Contents/Managed/UnityEngine.dll' could not be found

Compilation failed: 1 error(s), 0 warnings
UnityEngine.Debug:LogError(Object, Object)
CoreEditor.AssemblyPacker.Packer:CreateAssemblies() (at Assets/Core.AssemblyPacker/Editor/Packer.cs:222)
CoreEditor.AssemblyPacker.Packer:PackAssembly() (at Assets/Core.AssemblyPacker/Editor/Packer.cs:95)
CoreEditor.AssemblyPacker.PackerEditor:OnInspectorGUI() (at Assets/Core.AssemblyPacker/Editor/PackerEditor.cs:18)
UnityEditor.DockArea:OnGUI()
```

Mine is located at `/opt/Unity/Editor/Data/Managed/UnityEngine.dll` on ArchLinux.
```
$ uname -a
Linux zbookarch.local 4.8.13-1-ARCH #1 SMP PREEMPT Fri Dec 9 07:24:34 CET 2016 x86_64 GNU/Linux
```

Unity version 5.5.0xf3linux

--- a/Assets/Core.AssemblyPacker/Editor/Packer.cs
+++ b/Assets/Core.AssemblyPacker/Editor/Packer.cs
@@ -269,7 +269,7 @@ namespace CoreEditor.AssemblyPacker
 		
 		public static string GetFrameWorksFolder()
 		{
-			if (Application.platform == RuntimePlatform.WindowsEditor)
+			if (Application.platform == RuntimePlatform.WindowsEditor || Application.platform == RuntimePlatform.LinuxEditor)
 			{
 				return (Path.GetDirectoryName(EditorApplication.applicationPath) + "/Data/");
 			}

